pkg/api/highLevelApi.go

```diff
@@ -19,7 +20,7 @@ const (
 
 // CreateUserAndAgentWithAuthDelegation Create and register a user and an agent identities with user delegating authentication to the agent.
 //func CreateUserAndAgentWithAuthDelegation(resolverClient register.ResolverClient, userSeed []byte, userKeyName string, agentSeed []byte, agentKeyName string, delegationName string, userName string, agentName string, userPassword string, agentPassword string, overrideDocs bool) (userID register.RegisteredIdentity, agentID register.RegisteredIdentity, err error) {
-func CreateUserAndAgentWithAuthDelegation(resolverClient register.ResolverClient, opts *CreateUserAndAgentWithAuthDelegationOpts) (userID register.RegisteredIdentity, agentID register.RegisteredIdentity, err error) {
+func CreateUserAndAgentWithAuthDelegation(ctx context.Context, resolverClient register.ResolverClient, opts *CreateUserAndAgentWithAuthDelegationOpts) (userID register.RegisteredIdentity, agentID register.RegisteredIdentity, err error) {
```

Comment:
For really long definitions we could split them something like this (to stick to approx 120 chars):
```go
func CreateUserAndAgentWithAuthDelegation(
	ctx context.Context, resolverClient register.ResolverClient, opts *CreateUserAndAgentWithAuthDelegationOpts,
) (userID register.RegisteredIdentity, agentID register.RegisteredIdentity, err error) {
```

.. but on the other hand they were long already here and elsewhere so actually probably no point in changing anything right now. :thinking:

Reply:
some cleanup performed.

Reply:
[9531021](https://github.com/Iotic-Labs/iotics-identity-go/pull/46/commits/95310215e1787b86f6cf0277a092f35090ab24f5)